cmake/modules/platform/toolcfg/gnu.cmake

```diff
@@ -68,7 +70,7 @@ list(APPEND TR_COMPILE_OPTIONS
 # TR_CXX_COMPILE_OPTIONS are appended to CMAKE_CXX_FLAGS, and so apply only to
 # C++ file compilation
 list(APPEND TR_CXX_COMPILE_OPTIONS
-	-std=c++0x
+	# nothing
```

Comment:
Can you remove the TR_CXX_COMPILE_OPTIONS?

Reply:
done